swagger.yaml

```diff
@@ -0,0 +1,51 @@
+openapi: 3.0.1
+info:
+  title: Ainized-Style-Transfer 
+  description:  |
+    # This is an API server that uses deep learning to compose one image in the style of another image
```

Comment:
What about introducing what style-transfer does in terms of explaining itself rather than saying 'server or deep learning stuff'?

Reply:
Done!